lib/specifications/Specification.js

```diff
@@ -110,47 +110,49 @@ class Specification {
 
 	/* === Attributes === */
 	/**
-	 * Get the project name
+	 * Get the name of this specification
 	 *
 	 * @public
-	 * @returns {string} Project name
+	 * @returns {string} Specification name
 	 */
 	getName() {
 		return this._name;
 	}
 
 	/**
-	 * Get the project kind
+	 * Get the kind of this specification. For example <code>project</code> or <code>extension</code>
 	 *
 	 * @public
-	 * @returns {string} Project kind
+	 * @returns {string} Specification kind
 	 */
 	getKind() {
 		return this._kind;
 	}
 
 	/**
-	 * Get the project type
+	 * Get the type of this specification.
+	 * For example <code>application</code> or <code>library</code> in case of projects,
+	 * and <code>task</code> or <code>server-middleware</code> in case of extensions
 	 *
 	 * @public
-	 * @returns {string} Project type
+	 * @returns {string} Specification type
 	 */
 	getType() {
 		return this._type;
 	}
 
 	/**
-	 * Get the project Specification Version
+	 * Get the Specification Version
 	 *
 	 * @public
-	 * @returns {string} Project Specification Version
+	 * @returns {string} Specification Version
 	 */
 	getSpecVersion() {
 		return this._specVersion;
 	}
 
 	/**
-	 * Get the project version as typically defined in the project's <code>package.json</code>
+	 * Get the specification's version, as typically defined in a <code>package.json</code>
```

Comment:
Specification Version ?

Reply:
Not in this case. This is the generic version of the entity that makes up this "Specification instance".

Reply:
"Get the specification's generic version, as typically defined in a <code>package.json</code>"

Reply:
Done 👍